.github/workflows/publish.yml

```diff
@@ -33,14 +33,14 @@ jobs:
 
     - name: Bump react version patch
       run: |
-        grep -E "[0-9]+\.[0-9]+\.[0-9]+-beta\.[0-9]+" ./packages/react-icons/package.json | python3 -c """
+        grep -E "[0-9]+\.[0-9]+\.[0-9]+-rc\.[0-9]+" ./packages/react-icons/package.json | python3 -c """
         import sys
         import os
         current_version = sys.stdin.read().strip().split(':')[1]
         current_version = current_version.split('\"')[1]
-        major,minor,asset,beta = current_version.split('.')
-        asset = asset.replace('-beta','')
-        os.system(f'echo \"REACT_VERSION={major}.{minor}.{int(asset) + 1}-beta.{int(beta) + 1}\" >> \$GITHUB_ENV')
+        major,minor,asset,rc = current_version.split('.')
+        asset = asset.replace('-rc','')
+        os.system(f'echo \"REACT_VERSION={major}.{minor}.{int(asset) + 1}-rc.{int(rc) + 1}\" >> \$GITHUB_ENV')
```

Comment:
This has me thinking we ought to find a solution that more easily allows beta/rc/release, but that can sit on the back burner if you don't immediately think of a better solution